js/apps/system/_admin/aardvark/APP/react/src/views/collections/addCollection/AddCollectionForm.tsx

```diff
@@ -0,0 +1,170 @@
+        <Divider borderColor="gray.400" />
+        <Grid
+          templateColumns={"1fr 1fr"}
+          rowGap="5"
+          columnGap="3"
+          marginY="4"
+          maxWidth="full"
+        >
+          {window.App.isCluster && (
+            <>
+              <InputControl
+                isDisabled={
+                  isDisabled || isOneShard || distributeShardsLike.value !== ""
+                }
+                name={"numberOfShards"}
+                label="Number of shards"
+                inputProps={{
+                  type: "number",
+                  min: 1,
+                  max: window.frontendConfig.maxNumberOfShards
+                }}
+                tooltip={
+                  isOneShard
+                    ? undefined
+                    : `The number of shards to create. The maximum value is ${window.frontendConfig.maxNumberOfShards}. You cannot change this afterwards.`
+                }
+              />
+              <CreatableMultiSelectControl
+                isDisabled={isDisabled}
+                name={"shardKeys"}
+                label="Shard keys"
+                selectProps={{
+                  noOptionsMessage: () => "Start typing to add a key"
+                }}
+                tooltip={"The keys used to distribute documents on shards."}
+              />
+              <InputControl
+                isDisabled={
+                  isDisabled ||
+                  isSatellite.value ||
+                  distributeShardsLike.value !== ""
+                }
+                name={"replicationFactor"}
+                label="Replication Factor"
+                inputProps={{
+                  type: "number",
+                  min: window.frontendConfig.minReplicationFactor ?? 1,
+                  max: window.frontendConfig.maxReplicationFactor
+                }}
+                tooltip={"Total number of copies of the data in the cluster."}
+              />
+              <InputControl
+                isDisabled={
+                  isDisabled ||
+                  isSatellite.value ||
+                  distributeShardsLike.value !== ""
+                }
+                name={"writeConcern"}
+                label="Write Concern"
+                inputProps={{
+                  type: "number",
+                  min: 1,
+                  max: window.frontendConfig.maxReplicationFactor
+                }}
+                tooltip={
+                  "Total number of copies of the data in the cluster that are required for each write operation. If we get below this value the collection will be read-only until enough copies are created. Must be smaller or equal compared to the replication factor."
+                }
+              />
+              {window.frontendConfig.isEnterprise && (
+                <>
+                  {!isOneShard && (
+                    <SelectControl
+                      isDisabled={isDisabled}
+                      name={"distributeShardsLike"}
```

Comment:
Update: This is not required as the `isCoordinator` check is exactly the same as the `isCluster` check (because you can't access a non-coordinator via the web UI)